### ISSUE TITLE
Can't disable new year's fireworks

### ISSUE THREAD

**Issue by Rukachan** [SleepyTrousers/EnderCore#33]

**Comment by tterrag1098** [comment]
Ok? Is this really that big a deal? It happens once per year.

**Comment by Rukachan** [comment]
It ruins my 1st of Jan, so I would say yes.

**Comment by Vaygrim** [comment]
Actually when you've got some 9 mods that all fire this off without option for being disabled, it can get laggy loud and annoying VERY fast. EnderCore, Ender IO Addons, Mekanism, and NUMEROUS other mods were doing stuff like this last night and this morning and it makes it near impossible to record / get anything done.

**Comment by Turkey2349** [comment]
It's a feature that happens once per year, once per world and spawns about 7 fire works. This is hardly something that would ruin someone's day.

**Comment by Rukachan** [comment]
https://twitter.com/tterrag1098/status/683019234528604162
Yes, really
I don't think that asking your twitter army to harass whoever posts anything you don't agree with as something acceptable.

@Turkey2349 
I make many new worlds in a day and I have to deal with this never-ending firework launching from multiple mods with no option to disable.
And yes, it happens only once per year, this is why it ruins only one day per year.

**Comment by Vaygrim** [comment]
Another mod (Ender IO Addons) basically pulled this stunt as well, and didn't have a configuration for disabling it either. Trouble is that mod celebrated it every hour ON the hour.. plus several half hour intervals for 'half timezones'. The result was 24 hours of fireworks and laggy particle spam.

THAT screwup has basically put lots of people in a bad mood about stuff like this.

So I'd like to request very nicely: "Can we pretty please get settings like this put under a config option, so that we may disable them? I think this would be very nice of you."

**Comment by Turkey2349** [comment]
@Vaygrim sounds like you should take it up with that mod then

**Comment by Vaygrim** [comment]
@Turkey2349 I already have, but it would be nice to have options to disable things like this in ALL mods. I really don't think it is that 'bad' of a request to make. Yes the original poster might have proposed the request rather poorly .. but I find it to be a perfectly valid request in and of itself.

**Comment by tterrag1098** [comment]
My *twitter army*?? Literally no one from twitter has come here. And yes, I think you're being a bit ridiculous. Anyways, this is getting off subject. I might add a config but it won't be soon, not in time to do anything about this year.
